examples/remoteBuild.md

```diff
@@ -24,33 +42,21 @@ someone who wants to send them all at the same time):
 
 <!-- @remoteBuild @testAgainstLatestRelease -->
 ```
-target="https://github.com/kubernetes-sigs/kustomize/examples/multibases?ref=v1.0.6"
+target="https://github.com/kubernetes-sigs/kustomize//examples/multibases?ref=v1.0.6"
 test 3 == \
   $(kustomize build $target | grep cluster-a-.*-myapp-pod | wc -l); \
   echo $?
 ```
 
-The URL can be an archive
-
-<!-- @remoteBuild -->
-```
-target="https://github.com/kustless/kustomize-examples/archive/master.zip//kustomize-examples-master"
-test 1 == \
-  $(kustomize build $target | grep remote-cm | wc -l); \
-  echo $?
-```
-
-Note the kustomize root path inside archive must be appended after `//`.
-
-A base can be a URL:
+A remote kustomization directory resource can also be a URL:
```

Comment:
Any particular reason you prefer `remote kustomization directory resource` over `remote base`? The latter feels more concise and clear to me

Reply:
I thought `base` was deprecated, but I guess it's only the `bases` field but not the concept itself.